vision/unit_tests/test_module.py

```diff
@@ -363,10 +361,87 @@ def test_return(self):
         #   and arbitrary center value
         depth_image = np.zeros(IMAGE_SIZE, dtype=int)
         arbitrary_value = 300
-        roi = region_of_interest(depth_image, arbitrary_value, (arbitrary_value, arbitrary_value))
+        roi = region_of_interest(
+            depth_image, arbitrary_value, (arbitrary_value, arbitrary_value)
+        )
 
         self.assertIs(type(roi), np.ndarray)
 
 
+class TestModuleRoll(unittest.TestCase):
+    """
+    Testing module.get_module_roll for validity.
+    """
+
+    def test_params(self):
+        """
+        Verify can handle input types
+
+        Parameters
+        ----------
+        enclosing_region: ndarray
+            The depth image.
+        module_roll: np.float64
+            Calculated float angle.
+        """
+
+        IMAGE_SIZE = [1080, 1920, 3]
+
+        enclosing_region = np.zeros(IMAGE_SIZE, dtype=np.uint8)
+        module_roll = get_module_roll(enclosing_region)
+
+        self.assertIs(type(enclosing_region), np.ndarray)
+        self.assertIs(type(module_roll), np.float64)
+
+    def test_module_roll(self):
+        img_dir = os.path.join(gparent_dir, "vision_images/module/Feb29")
+        files = os.listdir(img_dir)
+
+        estimates = {
+            # FORMAT: "name_of_file" : [estimated float, (estimated center)]
+            "2020-02-29_15.36.15.167565": [20, (1180, 300)],
+            "2020-02-29_15.40.46.652448": [14, (700, 550)],
+            "2020-02-29_15.40.48.862847": [10, (1300, 630)],
+            "2020-02-29_15.40.40.444538": [5, (830, 670)],
+            "2020-02-29_15.40.54.564617": [8, (435, 650)],
+        }
+
+        for current_file in estimates.keys():
+            # Selects image
+            current_image_file = os.path.join(img_dir, current_file) + "-colorImage.jpg"
+            current_depth_file = os.path.join(img_dir, current_file) + "-depthImage.npy"
+            # Loads images
+            colorImage = cv2.imread(current_image_file)
+            depthImage = np.load(current_depth_file)
```

Comment:
One other small thing is that we are trying to move towards using snake case (instead of camel case) for everything except class names, so these should be changed.

Should be:
`color_image`
`depth_image`

Reply:
Looks like this is the only place you used camel case, so these are the only ones that need to be changed. (unless I missed some somewhere).